### DISCUSSION TITLE
Sound of Alarm not trigerred or late by 5 to 15 mns

### DISCUSSION THREAD

**Discussion by gaijin213** [FossifyOrg/General-Discussion#639] [General]
Hi,

I  using Macdroid to automatically set the device in silent/Dnd Mode and Airplane mode at night. And on may occasions my Clock does not get triggered or it does late aftrt 5-15mns and only when I turn on my screen manually.

I tried other Clock apps including stock Miui, Lineage one and other simple ones from F Droid and they trigger without issues. j also gave all possible permissions and removed any battery optimization for this app but is is sporadic and Inconsistent. 
Any thought please?

Thanks!

**Comment by Aga-C** [comment]
It seems to be a bug. Please fill the [bug report](https://github.com/FossifyOrg/Clock/issues/new?assignees=&labels=bug%2Cneeds+triage&projects=&template=bug_report.yml) including as many details as you can, so we could try to reproduce it.
